README.md

```diff
@@ -10,7 +10,7 @@ See this blog post for more info and how to use: http://ericeastwood.com/blog/8/
 
 The code is a bit crusty and old now but happy to have it on GitHub. It could also use a lot of optimization to make it faster, feel free to submit a PR.
 
+# Other projects
+- [Pedro, @pvieito](https://github.com/pvieito) made a CLI version of this app in Python that you can find in [this gist](https://gist.github.com/pvieito/8cdb54a9a03fd36e51c8df6e331a3006).
```

Comment:
Sorry, random markdown lints:

 - Let's add some white space between the heading and list items
 - I prefer a single space before the dash (example below)
 
```
 - foo
 - bar
 - baz
```